pkg/types/types.go

```diff
@@ -81,7 +82,7 @@ type RdmaDeviceSpec interface {
 	VerifyRdmaSpec([]*pluginapi.DeviceSpec) error
 }
 
-// PciNetDevice provides an interface to get generic device specific information
+// bkb provides an interface to get generic device specific information
```

Comment:
It looks like this is an accidental renaming.

Reply:
thanks. fixed